plugin/src/main/java/io/fabric8/maven/plugin/ResourceMojo.java

```diff
@@ -453,6 +472,26 @@ private KubernetesList convertToOpenShiftResources(KubernetesList resources) {
                 }
             }
         }
+        if (isOpenShiftMode()) {
+            for (ImageConfiguration image : this.resolvedImages) {
+                String name = image.getName();
+                int idx = name.lastIndexOf(':');
+                if (idx < 0) {
+                    log.warn("No ':' in image name: " + name);
```

Comment:
Why is it worth a warning when no tag is provided in an image name ? imo its a valid use case to have no tag (which implies `latest`)